src/tests/ftest/dfuse/daos_build.py

```diff
@@ -134,18 +118,34 @@ def test_daos_build(self):
                 'scons -C {} --jobs 24 --build-deps=only'.format(build_dir),
                 'scons -C {} --jobs {}'.format(build_dir, build_jobs)]
         for cmd in cmds:
-            try:
-                command = '{};{}'.format(preload_cmd, cmd)
-                # Use a 10 minute timeout for most commands, but vary the build timeout based on
-                # the dfuse mode.
-                timeout = 10 * 60
-                if cmd.startswith('scons'):
-                    timeout = build_time * 60
-                ret_code = general_utils.pcmd(self.hostlist_clients, command, timeout=timeout)
-                if 0 in ret_code:
-                    continue
-                self.log.info(ret_code)
-                raise CommandFailure("Error running '{}'".format(cmd))
-            except CommandFailure as error:
-                self.log.error('BuildDaos Test Failed: %s', str(error))
-                self.fail('Unable to build daos over dfuse in mode {}.\n'.format(cache_mode))
+            command = '{};{}'.format(preload_cmd, cmd)
+            # Use a short timeout for most commands, but vary the build timeout based on dfuse mode.
+            timeout = 10 * 60
+            if cmd.startswith('scons'):
+                timeout = build_time * 60
+            start = time.time()
+            ret_code = general_utils.run_pcmd(self.hostlist_clients, command, verbose=True,
+                                              timeout=timeout, expect_rc=0)
+            elapsed = time.time() - start
+            self.log.info('Ran in {} seconds\n'.format(elapsed))
+            assert len(ret_code) == 1
+
+            cmd_ret = ret_code[0]
+            self.log.info(cmd_ret)
+
+            self.log.info(cmd_ret['stdout'])
```

Comment:
This line duplicates the one above, and it prints it as expected being a python array of lines, so it's one very long line in the logfile.  This code needs to iterate through the array.